07-Checkout/readme.md

```diff
@@ -1,48 +1,53 @@
-# git-checkout
+# Working with branches
 
 * Switch branches or restore working tree files
 
 Let's create OS X version after all. This time we'll use `git-checkout`
 with `-b` switch, which creates new branch if it doesn't exist already
 and checkout the new branch
 
-* ```git checkout -b my_apple_app```
+```
+> git checkout -b my_apple_app
+Switched to a new branch 'my_apple_app'
+```
 
 With knowledge from last part we can check if it worked
 
-* ```git branch```
-
-
-```  master
+```
+> git branch
+master
 * my_apple_app
 ```
 
 We "implement" OS X version and commit it
 
-* ```mkdir MyAppleApp```
-* ```echo "OS X implementation" > MyAppleApp/osx.txt```
-* ```git add MyAppleApp```
-* ```git commit -m "OS X version"```
+* `mkdir MyAppleApp`
```

Comment:
If you already changed the formatting of the other commands, you should change these as well:
```
> mkdir MyAppleApp
> echo "OS X implementation" > MyAppleApp/osx.txt
> git ...